### PR TITLE
Added support for www.emmikochteinfach.de

### PR DESCRIPTION
**Pull request by derteufelqwe**
I have added a the scraper for www.emmikochteinfach.de
The receipe in the test is https://emmikochteinfach.de/klassisches-rindergulasch/

### CODE REVIEW

**Comment by jayaddison** [comment]
Hi @derteufelqwe - this looks good, thank you!

There are some lint check failures which can be fixed by running the [`black`](https://github.com/psf/black/) code formatter on your code.

Also since this opened, we have merged #601 which added support for `mypy` Python type checks.  If you can pull and merge the latest changes from the `main` branch into this one, then those will run before we integrate your changes.

**Comment by derteufelqwe** [comment]
I somehow forgot I wanted to update this ;P
This should fix the issues.

File: recipe_scrapers/emmikochteinfach.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from ._abstract import AbstractScraper
```
Comment (jayaddison) [inline recipe_scrapers/emmikochteinfach.py:1]:
```suggestion
# mypy: allow-untyped-defs

from ._abstract import AbstractScraper
```

Adding this to the top of the file should resolve the `mypy` check warnings.  Looks like I forgot to update the scraper template to add this..

**Review by jayaddison — APPROVED** [review]
Looks good, thank you @derteufelqwe

**Pull request merged**